### PR TITLE
Update zh.dictionary.itop.core.php

### PR DESCRIPTION
**Pull request by purplegrape**

### CODE REVIEW

**Comment by purplegrape** [comment]
my very first pull request，I will mainly contribute to chinese language（zh_CN） translation including iTop 2.4 +

**Comment by piRGoif** [comment]
Hello, the same file is included in your changes in #24... Are there some overlapping modifications ?

File: dictionaries/zh.dictionary.itop.core.php [hunk 1]
```diff
@@ -413,12 +413,12 @@
 	'Core:AttributeBoolean' => 'Boolean~~',
 	'Core:AttributeBoolean+' => 'Boolean~~',
 	'Core:AttributeBoolean/Value:null' => '',
-	'Core:AttributeBoolean/Value:yes' => 'Yes~~',
-	'Core:AttributeBoolean/Value:no' => 'No~~',
+	'Core:AttributeBoolean/Value:yes' => '是~~',
+	'Core:AttributeBoolean/Value:no' => '否~~',
```
Comment (piRGoif) [inline dictionaries/zh.dictionary.itop.core.php]: Warning, the leading "~~" are present only to indicate that this is a new key that needs a translation (see https://www.itophub.io/wiki/page?id=2_5_0%3Acustomization%3Adatamodel#localization), you should remove them after translation is done.

Comment (purplegrape) [reply dictionaries/zh.dictionary.itop.core.php]: got it，many thnks

**Pull request closed without merging**